src/renderers/webgl/WebGLState.js

```diff
@@ -317,7 +317,7 @@ function WebGLState( gl, extensions, utils ) {
 	var enabledAttributes = new Uint8Array( maxVertexAttributes );
 	var attributeDivisors = new Uint8Array( maxVertexAttributes );
 
-	var capabilities = {};
+	var enabledCapabilities = {};
```

Comment:
Sorry for the comment after merging.

English question. Is this var name `enabledCapabilities` good? `WebGLState` already had `capabilities` so I wanted to rename because I wanna use `capabilities` for `WebGLCapabilities`.

Reply:
Yeah. I think it's okay 🙂